OneCommunityGlobal/HighestGoodNetworkApp
# Lucile - ForcePassword and Reports tests

# Description
Unit Tests HGN

<google-sheets-html-origin>

ForcePasswordUpdate | ForcePasswordUpdate.jsx | Yes. Have test files in two places. Need to reconcile together.https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/blob/bc16494d8f4a3bdb371d5d86374cbd5522bf2642/src/__tests__/ForcePasswordUpdate.test.jshttps://github.com/OneCommunityGlobal/HighestGoodNetworkApp/blob/bc16494d8f4a3bdb371d5d86374cbd5522bf2642/src/components/ForcePasswordUpdate/ForcePasswordUpdate.test.jsx
-- | -- | --




<google-sheets-html-origin>

Reports | Reports.jsx | there is an empty test file
-- | -- | --


## Main changes explained:
- merge ForcePassword test files and create Reports test file

## How to test:
1. check into current branch
2. do `npm test` 
3. verify that ForcePassword & Reports tests PASS
